push_docker.sh

```diff
@@ -3,40 +3,76 @@ REGISTRY_SERVER=$2
 REGISTRY_PORT=$3
 REGISTRY_USERNAME=$4
 REGISTRY_PASSWD=$5
-DOCKER_IMAGE_TAG=$6
+DOCKER_IMAGE_TAG=""
+PLATFORM=""
+SONIC_VERSION=""
+
+shift 5
+
+if [ "$#" != "0" ]
+then
+    DOCKER_IMAGE_TAG=$1
+    shift
+fi
+
+if [ "$#" != "0" ]
+then
+    PLATFORM=$1
+    shift
+fi
+
+if [ "$#" != "0" ]
+then
+    SONIC_VERSION=$1
```

Comment:
>SONIC_VERSION [](start = 4, length = 13)

What if user only have SONIC_VERSION but no PLATFORM?

Reply:
This script does not support it. Being positional args user can put "generic", when PLATFORM can't be given.